content/references/gamification/privileges.md

```diff
@@ -0,0 +1,23 @@
+---
+kind: reference
+sidebar: ranks-honor
+prev: /references/gamification/honor/
+---
+
+# Privileges
+
+| Privilege                           | Required Honor | Description                                                  |
+| ----------------------------------- | -------------: | ------------------------------------------------------------ |
+| Vote Kata                           |             25 | Vote on how satisfied you were with a kata                   |
+| Mark comment as spoiler             |             50 | Mark another's comment as a spoiler                          |
+| Estimate ranking of own beta kata   |             75 | Estimate on what rank you think your beta kata should be     |
+| Assess Rank                         |            100 | Vote on what rank you think a beta kata should be            |
+| Create Kata                         |            300 | Contribute your own kata to the community                    |
+| Unmark comment as spoiler           |            500 | Unmark another's comment as being a spoiler                  |
+| Vote 2x power                       |          1,000 | Your vote counts 2x towards getting a kata out of beta       |
+| Vote 3x power                       |          2,000 | Your vote counts 3x towards getting a kata out of beta       |
+| Vote 4x power                       |          3,000 | Your vote counts 4x towards getting a kata out of beta       |
+| Coauthor Kata / Approve Translation |          4,000 | Ability to edit other author's kata and approve translations |
+| Resolve comment                     |          5,000 | Ability to resolve other's comments                          |
+| Approve Kata                        |          6,000 | Ability to approve a beta kata and assign its rank           |
+| Edit locked test cases              |         10,000 | Ability to edit test cases when they're locked               |
```

Comment:
```suggestion
| Edit locked test cases              |         10,000 | Ability to edit test cases when they're locked               |


**†** You can edit a kata created by others if:

- You have contributed to the kata in the past (you do not need to have a coauthoring privilege in this case) **OR**
- You have been granted coauthoring privilege **and** at least one of the following are true:
  - The kata is in beta and waiting for approval
  - The kata is in beta and the author have allowed contribution
  - The kata is approved and unresolved issue/suggestion created over a week ago exists
```

I think you just missed my first comment:

> Move these under the table because it doesn't make sense on this page.

So adding it back.